### PR TITLE
clean up

### PR DESCRIPTION
**Pull request by leviboenish**
nice 😎

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,9 @@
...
-        if (node.nodeType === 3) {
-            let text = node.nodeValue;
-            let replaced = text.replace('69', 'Nice 😎');
-
-            if (replaced !== text) {
-                node.nodeValue = replaced;
-            }
+const elements = document.getElementsByTagName('*');
+const elementsArray = Array.from(elements);
+elementsArray.forEach(element => {
```
Comment (xdesro) [inline index.js]: I might do `[...elements].forEach()` but that's literally just stylistic.

File: index.js [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
+    element.childNodes.forEach(node => {
+        if (node.nodeType === 3 && node.nodeValue.includes('69')) {
+            element.innerHTML = node.nodeValue = node.nodeValue.replace('69', '69 <-- Now <strong>That\'s</strong> Nice 😎');
```
Comment (xdesro) [inline index.js]: How does this affect elements that just mention 69 in a large block of text? (Testing currently)

**Review by xdesro — APPROVED** [review]
Non-blocking, just some notes. Lookin' hella fresh.

File: index.js (hunk 2 above)
Comment (xdesro) [reply index.js]: Never mind, the screenshot in Telegram clears this up. Attaching for posterity.

IMAGE 2020-01-02 12:44:25

Comment (xdesro) [inline index.js]: Could we use something like `⇦` instead of `<--`?

Comment (leviboenish) [reply index.js]: which part are you referring to?

Comment (leviboenish) [reply index.js]: yes

File: index.js (hunk 1 above)
Comment (leviboenish) [reply index.js]: this is hot

**Pull request merged**